teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorOptions.java

```diff
@@ -139,10 +139,9 @@ public class ValidatorOptions {
   private int executorThreads = DEFAULT_VALIDATOR_EXECUTOR_THREADS;
 
   @Option(
-      names = {"--Xblock-v3-enabled"},
+      names = {"--block-v3-enabled"},
```

Comment:
i think leaving it as a dev option thats just enabled by default was what i was thinking, i am hoping that this flag disappears once we prove we don't require it...